stdlib/experimental/http/get_test.go

```diff
@@ -0,0 +1,94 @@
+package http_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux"
+	_ "github.com/influxdata/flux/builtin"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/dependencies/url"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func addFail(scope values.Scope) {
+	scope.Set("fail", values.NewFunction(
+		"fail",
+		semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+			Return: semantic.Bool,
+		}),
+		func(ctx context.Context, args values.Object) (values.Value, error) {
+			return nil, errors.New(codes.Aborted, "fail")
+		},
+		false,
+	))
+}
+
+func TestGet(t *testing.T) {
+	var req *http.Request
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, request *http.Request) {
+		req = request
+		var err error
+		if err != nil {
+			w.WriteHeader(500)
+			w.Write([]byte(err.Error()))
+		}
+		w.WriteHeader(204)
+	}))
+	defer ts.Close()
+
+	script := fmt.Sprintf(`
+import "experimental/http"
+
+status = http.get(url:"%s/path/a/b/c", headers: {x:"a",y:"b",z:"c"})
```

Comment:
Would be better if this returned a response object that had `status` and `headers` and `body`. Also, would be nice to be able to pass in `urls` that can be fetched in parallel by Go. The return for that would be an array of responses, where the position in the array is the same as the position in the passed in the urls.

Reply:
Also would be good to have a `timeout` parameter that is a duration. So you could have `5s` or `125ms` or whatever.

Reply:
Oh, would also be nice to have `time` as a duration that can ideally be converted into an int in milliseconds or whatever.

Reply:
Actually, now that I think of it, it would probably be better to keep `Get` consistent for a single url. But add another function named `GetAll` that takes an array of urls and returns an array of responses.